game/player.py

```diff
@@ -5,10 +5,14 @@
 
 
 class Player:
+    """Holds current information about the player"""
+
     team: List[Pokemon] = []
 
     def __init__(self):
-        if (len(self.team) < 1):
+        """fills it with a random Pokemon from PokeApi"""
+
+        if len(self.team) < 1:
             self.team.append(pokeapi.get_random_pokemon_from_api())
 
 
```

Comment:
For `PLAYER`: Don't forget to document global variables.

Reply:
Added the part that these player variable is saved as a global variable

Reply:
Minor: don't capitalize references to variables.
`Fills the player's team with a random Pokemon from PokeApi`
The Player class doesn't actually do any of the saving to the global PLAYER variable, so I wouldn't include that.